testing/run-scripts/run_cloud.sh

```diff
@@ -153,15 +151,11 @@ HOST_IP=`ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1`
 
 # Start Zork, if necessary.
 if ! docker ps -a | grep uproxy-zork >/dev/null; then
-  if ! docker images | grep uproxy/$1 >/dev/null; then
-    BROWSER=$(echo $1 | cut -d - -f 1)
-    VERSION=$(echo $1 | cut -d - -f 2)
-    IMAGEARGS=
-    if [ -n "$PREBUILT" ]
-    then
-      IMAGEARGS="-p"
-    fi
-    ${BASH_SOURCE%/*}/image_make.sh $IMAGEARGS $BROWSER $VERSION
+  if ! docker images | grep $IMAGE >/dev/null; then
```

Comment:
Seems like we should remove this, right? If the image doesn't exist locally then Docker will try to pull it from Docker Hub - which is what we now want :-)